pylint/checkers/base/basic_error_checker.py

```diff
@@ -145,7 +145,7 @@ class BasicErrorChecker(_BasicChecker):
             "pre-decrement operator -- and ++, which doesn't exist in Python.",
         ),
         "E0108": (
-            "Duplicate argument name %s in function definition",
+            "Duplicate argument name %r in function definition",
```

Comment:
This is an incidental change to add quotes to the name of the missing argument in the message output.